rest-server/src/templates/dockerContainerScript.mustache

```diff
@@ -46,20 +50,27 @@ export PAI_KILL_ALL_ON_COMPLETED_TASK_NUM={{{ jobData.killAllOnCompletedTaskNumb
 
 export PAI_CURRENT_CONTAINER_PORT=$(( (RANDOM % 55535) + 10001 ))
 hdfs dfs -touchz $HDFS_LAUNCHER_PREFIX/$PAI_USERNAME/$PAI_JOB_NAME/tmp/$PAI_TASK_ROLE_NO-$PAI_TASK_INDEX-$PAI_CURRENT_CONTAINER_IP-$PAI_CURRENT_CONTAINER_PORT || exit 1
-while [ `hdfs dfs -ls $HDFS_LAUNCHER_PREFIX/$PAI_USERNAME/$PAI_JOB_NAME/tmp/ | wc -l` -lt  $PAI_TASKS_NUM ]; do
+while [ `hdfs dfs -ls $HDFS_LAUNCHER_PREFIX/$PAI_USERNAME/$PAI_JOB_NAME/tmp/ | grep "/$PAI_JOB_NAME/tmp" | wc -l` -lt  $PAI_TASKS_NUM ]; do
+  printf "%s\n%s\n\n" "[INFO]" "Waiting for other containers ..."
```

Comment:
>printf "%s\n%s\n\n" "[INFO]" "Waiting for other containers ..." [](start = 2, length = 63)

It will generate repeated log lines like this:

[INFO]
Waiting for other containers ...

[INFO]
Waiting for other containers ...

It looks this is too obverious log page to be non user logs, we can think about shrink lines it occupied, suggest to put [INFO] and "Waitting..." messages the same line and remove the extra blank line.

Reply:
updated